NEWS

```diff
@@ -6316,6 +6318,9 @@ CHANGES WITH 210:
           IFUNC. Please make sure to use --enable-compat-libs only
           during a transitional period!
 
+        * The .include syntax has been deprecated and is not documented
+          anymore. Drop-in files in .d directories should be used instead.
```

Comment:
i would probably have written `.d/` rather than just `.d`, following our logic of suffix dirs with /, but I guess this works too.